index.html

```diff
@@ -5,8 +5,150 @@
+
+    <hr>
+
+        <section>
+          <h2>Мы — маленькая, но гордая<br> дизайн-студия из Краснодара.</h2>
+            <p>Исповедуем принципы минимализма и чистоты. Ставим математический расчет<br>
+              превыше креатива. Работаем не покладая рук, как роботы.</p>
+          <h3>Выполняем заказы на разработку:</h3>
+              <ul>
+                <li>Веб-сайтов любой сложности</li>
+                <li>Мобильных приложений для iOS и Android</li>
+                <li>Слайдшоу и видео для корпоративных презентаций</li>
+              </ul>
+        </section>
+
+        <section>
+          <img src="https://via.placeholder.com/360x208" width="360" height="208" alt="">
+            <table>
+              <caption>с 2004 года Любим точность во всем:</caption>
+                <tr>
+                  <td>146%</td>
+                  <td>100%</td>
+                  <td>50%</td>
+                </tr>
+                <tr>
+                  <td>Уровень самоотдачи</td>
+                  <td>Соблюдение сроков</td>
+                  <td>Минимальная предоплата</td>
+                <tr
+            </table>
+        </section>
+
+    <hr>
+
+        <section><!--Logo-->
+            <img src="https://via.placeholder.com/199x57" width="199" height="57" alt="">
+            <img src="https://via.placeholder.com/209x90" width="209" height="90" alt="">
+            <img src="https://via.placeholder.com/185x52" width="185" height="52" alt="">
+            <img src="https://via.placeholder.com/173x84" width="173" height="84" alt="">
+        </section>
+
+    <hr>
+        <section><!--Карта+контакты+КартинкаМаркер-->
+          <img src="https://via.placeholder.com/1400x416" width="1400" height="416" alt=""> 
+          <section><!--Контакты-->
+            <h3>NЁRDS DESIGN STUDIO</h3>
+              <p>наб. Реки Карповки,<br>
+                5 лит П, Санкт-Петербург<br>
+                <a href="tel:+78005558628">+7 800 555-86-28</a>
+              </p>
+              <button type="button" name="msg_btn">Напишите нам</button>
+          </section>
+          <figcaption><!--Map marker-->
+            <img src="https://via.placeholder.com/231x190" width="231" height="190" alt="">
+          </figcaption>
+        </section>
+
+        <section><!--POPUP-->
+          <h2>Напишите нам</h2>
+            <button type="button">Закрыть</button>
+              <form action="https://echo.htmlacademy.ru" method="post">
+                <label for="name-name">Ваше имя:</label>
+                  <input id="name-name" type="name" name="name" placeholder="Имя Фамилия">
+                <label for="e-mail">Ваш email:</label>
+                  <input id="e-mail" type="email" name="e-mail" placeholder="email@example.com">
+                <label for="text">Текст письма:</label>
+                  <textarea id="message" class="txtarea" name="message" placeholder="В свободной форме:"></textarea>
+            <button id="send">Отправить</button>
```

Comment:
```suggestion
            <button id="send" type="submit">Отправить</button>
```
Нужно явно указывать тип кнопки. И хотя значение атрибута `type` по умолчанию равно `submit`, явное всегда лучше неявного.